.readthedocs.yaml

```diff
@@ -0,0 +1,26 @@
+# .readthedocs.yaml
```

Comment:
I added this config file because RTD was trying to use Python 3.7 by default and failing.